Dashboard/ViewModels/ScrutineeringViewModel.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Dashboard.Models;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Dashboard.ViewModels;
+
+public class ScrutineeringViewModel : ViewModelBase, IDisposable
+{
+    private readonly IDataStore _dataStore;
+    private readonly FileSystemWatcher _fileWatcher;
+    private YamlData _yamlData = new();
+
+    public ScrutineeringViewModel(IDataStore dataStore)
+    {
+        // This constructor is used for design-time data, so we don't need to start the connector
+        _dataStore = dataStore;
+        _dataStore.DataUpdated += OnDataChanged;
+
+        // The folder we are in at runtime is net8.0 (AV-Datalogger/Dashboard/bin/Debug/net8.0/Dashboard.exe), as
+        // our yaml file is in the resources folder, exit the current folder three times
+        var pathDirectory = "../../../Resources/";
```

Comment:
we cant use this pathing, there is no guarantee that this is what the folder structure will be like. you need to add the yaml files to the build output and change this path

Reply:
600b9b93d346c00a344b21679828690179d39c07